modules/cardano-node.nix

```diff
@@ -44,6 +44,7 @@ let
        then "--bitcoin-distr \"${distributionParam}\""
        else "--flat-distr \"${distributionParam}\""))
     (optionalString cfg.jsonLog "--json-log ${stateDir}/jsonLog.json")
+    (optionalString (cfg.ekgSink != null) "--metrics +RTS -T -RTS --statsd-server ${cfg.ekgSink}")
```

Comment:
@avieth can you incorporate this into your PR at https://github.com/input-output-hk/cardano-sl/pull/368